Home Work 12 (JS advanced. DOM, BOM)

GeorgyChirkov opened the pull request:
![task-01](https://user-images.githubusercontent.com/55555620/68542833-71bcee00-03c1-11ea-8b41-3d85aa3364df.gif)

Commit:
complete: task-01

Commit:
fixminor: delete unnecessary spaces

Commit:
feat: complete task-02

aneelia commented on src/ex12_js_dom/icons/back.svg:
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
пропусти svg через какой-нить онлайн нормалайзер

GeorgyChirkov replied:
Сделал

aneelia commented on src/ex12_js_dom/task-01.js:
```diff
@@ -0,0 +1,40 @@
+    let gall = document.querySelector(".gallery");
+    let image = document.createElement("img");
+    let btnnext = document.querySelector("[class='btn btnnext']");
+    let btnprev = document.querySelector("[class='btn btnprev']");
```
плохое название и btnnext не лучше

aneelia replied:
ну и тут как раз отлично подойдут id вместо классов

GeorgyChirkov replied:
Поменял названия в js, class поделил на class и id

aneelia commented on src/ex12_js_dom/task-01.js:
```diff
@@ -0,0 +1,40 @@
+    image.setAttribute("style", styles);
+    gall.append(image);
+
+    function nextImage (){
```
функции - глаголы

GeorgyChirkov replied:
Поменял

aneelia commented on src/ex12_js_dom/task-01.js:
```diff
@@ -0,0 +1,40 @@
+        if (i === imgarray.length) {
+            i = 0;
+        }
+        image.setAttribute("src", "./asset/"+imgarray[i]);
```
ей богу, ты че как будто первый раз переменные объявляешь

GeorgyChirkov replied:
Поменял

aneelia commented on src/ex12_js_dom/task-01.js:
```diff
@@ -0,0 +1,40 @@
+    btnnext.addEventListener("click", nextImage);
+    document.addEventListener("keydown", (event) => {if (event.key === "ArrowRight") {nextImage()}});
+    btnprev.addEventListener("click", prevImage);
+    document.addEventListener("keydown", (event) => {if (event.key === "ArrowLeft") {prevImage()}});  
```
key === "ArrowLeft" это из ES2020?

GeorgyChirkov replied:
Как оказалось нет. Это уже используется и считается лучшим вариантом https://developer.mozilla.org/ru/docs/Web/API/KeyboardEvent

aneelia commented on src/ex12_js_dom/task-02.css:
```diff
@@ -0,0 +1,111 @@
+main {
+    flex: 1 0 auto;
+}
+/* header */
```
комменты зачем, тут же и так все понятно. а ты обновлял эту версию канбана той, что делал ранее?

GeorgyChirkov replied:
Удалил. В следующих домашках брал одну из старых версий, когда какая-то из новых будет смержена, то поправлю.

Commit:
fix: images, names, JS styles

normalized svg images
fix names of JS variables
change js styles into variables with backtik
